### PR TITLE
Show command output

### PR DESCRIPTION
**Pull request by jloleysens**
Mainly a fix to always show relevant command output when appropriate. For example, when listing all credential definitions.

Also updated contributing documentation with reasoning for when to use what type of logging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,14 +75,18 @@ We highly recommend using `environment`s to avoid the repetitive task of
 specifying agent URLs. With this CLI you can get up and running with our
 community agent as your default environment, or use your own agent.
 
+If you are getting started with the tool, we recommend enabling informational logs by
+passing the `--verbose` (or `-v`) flag to get more information about what the CLI is
```
Comment (blu3beri) [inline README.md:79]: We should mention that multiple "verbose" are allowed.

File: docs/contributing.md [hunk 2]
```diff
@@ -39,4 +39,22 @@ We love tests, but recognize that there is a shortage of them at the moment. We
 encourage you to take a look at [Rust's guide](https://doc.rust-lang.org/book/ch11-01-writing-tests.html) on how to create an automated test. We are happy
 to provide support for writing tests on the PR.
 
-Currently a simple suite of tests can be executed by running `./tests/run.sh`.
\ No newline at end of file
+Currently a simple suite of tests can be executed by running `./tests/run.sh`.
+
+
+### Using log levels
```
Comment (blu3beri) [inline docs/contributing.md:45]: Ah I see you check it here. Maybe refer to this section from above?

**Review by blu3beri — APPROVED** [review]
Nice additions. Did you test it all with ./tests/run.sh and some manual stuff?

**Comment by jloleysens** [comment]
@blu3beri I did both :)

The last output of the tests/run.sh was `error: {"connection_id": ["Not a valid UUID."]}`

Is this expected?

**Comment by blu3beri** [comment]
> @blu3beri I did both :)
> 
> 
> 
> The last output of the tests/run.sh was `error: {"connection_id": ["Not a valid UUID."]}`
> 
> 
> 
> Is this expected?

Mine does not do that. It means that the  default output is not the connection id. U added a prefix to the print (connection id) in green. Couls you print that as an info?

**Comment by jloleysens** [comment]
OK, i tested checking out the main branch (from where this branch started) and it looks like I still get that output.

**Comment by jloleysens** [comment]
I'm going to go ahead and merge so long, happy to take on debugging tests in a follow up

**Pull request merged**